ynr/apps/elections/uk/templatetags/home_page_tags.py

```diff
@@ -125,8 +125,11 @@ def by_election_ctas(context):
     )
 
     if context["SHOW_BY_ELECTION_CTA"]:
+        dates_to_ignore = getattr(settings, "SCHEDULED_ELECTION_DATES", [])
```

Comment:
Should this PR also add a `SCHEDULED_ELECTION_DATES` setting, or is that something you're just going to do in the deploy repo only?

Reply:
Yeah, I'll leave it up to this: https://github.com/DemocracyClub/YNR_deploy/commit/c6c7ba4ca7bef8f099d0632df6e4e656fd5748ec